### PR TITLE
Updates to EditorConfig Docs

### PR DESCRIPTION
**Pull request by kuhlenh**
** do not merge yet **

Making edits based on @pilchie and @sharwell comments. Still need to verify "supported versions" and default values for formatting.

### CODE REVIEW

**Comment by kuhlenh** [comment]
@pilchie @sharwell @dpoeschl here is the new PR based on comments so far

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ide/editorconfig-code-style-settings-reference.md](https://github.com/kuhlenh/visualstudio-docs/blob/patch-2/docs/ide/editorconfig-code-style-settings-reference.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C4%5C692e8138-f1b0-9929-d89e-225d116f9c25%5CPullRequest%5C201708041823036519-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kuhlenh** [comment]
Ok, I finished updating my bits. @Pilchie can you make sure this gets merged before 15.3 after you and Sam and David review it? Thanks!

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ide/editorconfig-code-style-settings-reference.md](https://github.com/kuhlenh/visualstudio-docs/blob/patch-2/docs/ide/editorconfig-code-style-settings-reference.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C4%5C692e8138-f1b0-9929-d89e-225d116f9c25%5CPullRequest%5C201708042209018967-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/ide/editorconfig-code-style-settings-reference.md [hunk 1]
```diff
@@ -1125,7 +1236,7 @@ The following are valid options for the severity of a naming style rule
 
  `none` and `silent` are synonymous and mean that no indication of any kind should be shown to the user. This has the effect of disabling this rule.
 
- `suggestion` means that the user will be shown the following in the errorlist: and the following in the IDE. The `suggetion` severity will allow the naming rule to run, but it will not cause the build to break.
+ `suggestion` means that the user will be shown the following in the Error List: and the following in the IDE. The `suggetion` severity will allow the naming rule to run, but it will not cause the build to break.
```
Comment (Pilchie) [inline docs/ide/editorconfig-code-style-settings-reference.md]: Spelling `suggetion`

File: docs/ide/editorconfig-code-style-settings-reference.md [hunk 2]
```diff
@@ -94,12 +94,15 @@ error | When this style is not being followed, show a compiler error.
...
+        - [Prefer braces](#prefer_braces)
 
 ## <a name="this_and_me">"This." and "Me." Qualification</a>
 ### <a name="this_and_me_fields">Fields (IDE0003/IDE0009)</a>
-|  Option Name | `dotnet_style_qualification_for_field` |
-| ------------- |:-------------:|
-| **Applicable Languages** | C# and Visual Basic
+| **Option Name** | **Applicable Languages** | **Visual Studio Default** | **Supported Version** |
+| ----------- | -------------------- | ----------------------| ----------------  |
+|  `dotnet_style_qualification_for_field` | C# and Visual Basic | false:none | Visual Studio 2017 RTW |
```
Comment (Pilchie) [inline docs/ide/editorconfig-code-style-settings-reference.md]: I wonder if we should call it 15.0 instead of RTW here.

Comment (Pilchie) [inline docs/ide/editorconfig-code-style-settings-reference.md]: @ghogen - is there a convention for referring to Visual Studio 2017 updates in the docs that this should follow?

**Comment by Pilchie** [comment]
Ping @sharwell, @jmarolf, @cyrusnajmabadi, @dpoeschl, for some more reviews

@ghogen - Barring feedback, who can get this merged?

File: docs/ide/editorconfig-code-style-settings-reference.md (hunk 2 above)
Comment (mikeblome) [inline docs/ide/editorconfig-code-style-settings-reference.md]: Updates are worded like this: Visual Studio 2017 version 15.3. But I think for the table, abbreviating "version" to "v." as you have done in other changes is fine.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ide/editorconfig-code-style-settings-reference.md](https://github.com/kuhlenh/visualstudio-docs/blob/patch-2/docs/ide/editorconfig-code-style-settings-reference.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C8%5C692e8138-f1b0-9929-d89e-225d116f9c25%5CPullRequest%5C201708082226513844-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**